api/envoy/extensions/wasm/v3/common.proto

```diff
@@ -0,0 +1,152 @@
+syntax = "proto3";
```

Comment:
`api/envoy/extensions/wasm` is the `WasmService` extension. The common definitions should be moved to `api/envoy/extensions/common/wasm/v3/wasm.proto` instead.

Reply:
I found that keeping same proto package name `envoy.extensions.wasm.v3` while moving to `envoy/extensions/common/wasm/v3/wasm.proto` is not possible (looks like the formatter or something else trying to delete the package). IIUC, renaming the package name would result in a breaking change so I'm stuck in a deadlock :disappointed: